_stbt/types.py

```diff
@@ -72,6 +72,10 @@ def intersect(cls, *args: RegionT) -> RegionT:
                 return None
         return Region.from_extents(*out)
 
+    @typing.overload
+    def bounding_box(cls, *args: Region) -> Region:
+        ...
```

Comment:
I think you'll need a second overload. I don't think the type checker will take the real function into account if an overload is present.

Reply:
Fixed.